src/pages/login/login.component.js

```diff
@@ -148,9 +152,18 @@ const SignIn = () => {
           </Button>
           <Grid container>
             <Grid item xs>
-              <Link href='#' variant='body2'>
+              <Box variant='body2' className={classes.link}>
                 Esqueceu sua senha?
-              </Link>
+              </Box>
+            </Grid>
+            <Grid item>
+              <Box
+                variant='body2'
+                className={classes.link}
+                onClick={() => history.push('/signup')}
+              >
+                Cadastre-se
+              </Box>
```

Comment:
In order to improve accessibility, this `Box` should be a link:
```suggestion
              <Link to='/signup'>
                <Box variant='body2' className={classes.link}>
                  Cadastre-se
                </Box>
              </Link>
```